Managers/FileManager.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace ThirdTeam_Study.Managers
+{
+    public class FileManager 
+    {
+        public string FilePath { get; }
+        public readonly JsonSerializerSettings settings;
+        
+        public FileManager(string path = "/Users/volodimir/ThirdTeam_Study/Files/Tutor.json")
```

Comment:
This will broke code on any other computer except yours

Reply:
I have fixed it, so it will work correctly